src/components/Comments.jsx

```diff
@@ -19,12 +20,12 @@ export const Comments = ({loggedIn}) => {
        }).catch((err) => {
         setLoading('error')
        })
-      }, []);
+      }, [commentRemoved]);
```

Comment:
I don't want you to re-render the comments after removal - I want you to filter it locally instead